### DISCUSSION TITLE
Profile context action

### DISCUSSION THREAD

**Discussion by BeerRider** [microsoft/terminal#12645] [Q&A] [answered]
Hello.

Is it possible to create actions specific for selected profile?
Or disable action for specific profile?

I want to disable WT action **_paste_** (Ctrl+V) for powershell profile(s) so that pasting would be handled by psreadline.

**Comment by zadjii-msft** [comment, marked as answer]
Unfortunately, this isn't yet possible. This is a feature we're tracking in #5790

**Reply by BeerRider** [reply]
Thank you for reply.
Will wait.
